specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
There is something wrong with the swagger. None of the models are getting rendered by editor.swagger.io.